RecoLocalTracker/SiPixelRecHits/src/PixelCPEFast.cc

```diff
@@ -103,14 +109,32 @@ void PixelCPEFast::fillParamsForGpu() {
   uint32_t oldLadder = 0;
   float rl = 0;
   float zl = 0;
-  float miz = 90, mxz = 0;
+  float miz = 500, mxz = 0;
```

Comment:
As far as I understand, `miz` could even be initialized to 0 or some other meaningless value here